### ISSUE TITLE
Cliffs in wrong direction

### ISSUE THREAD

**Issue by FTno** [Maproom/qmapshack#511]
### Describe the bug
[comment]: # (A clear and concise description of what the bug is)

QMapShack displays cliffs the wrong way, pointing in the wrong direction in the map linked to below.

### What have you done to circle down the problem?
[comment]: # (Did you compare to previous versions? What versions? Did you check if others have the same problem?)

Checked in another application.

### To Reproduce
[comment]: # (List the steps to reproduce the behavior)

1. Download this map: https://download.freizeitkarte-osm.de/garmin/latest/ESP_CANARIAS_en_gmapsupp.img.zip
2. Load the map
3. Find a cliff (for example the crater of Teide Tenerife)
4. See the cliff pointing in the wrong direction

### Expected behavior
[comment]: # (A clear and concise description of what you expected to happen)

Cliffs pointing in the correct direction.

### Screenshots
[comment]: # (If applicable, add screenshots to help explain your problem)
QMapShack:
![qms](https://user-images.githubusercontent.com/16176811/163543345-0e4c34d8-ea06-487a-9be8-b99a68b70e67.png)

From openstreetmap.org:
![openstreetmap org](https://user-images.githubusercontent.com/16176811/163543566-ba8e40f4-961e-4081-b1f4-0f9b83515f6a.png)

### Attachments
[comment]: # (Can you provide an example file to trigger the problem?)

### Tracebacks
[comment]: # (Add your backtrace below if you have one. If QMapshack crashes the fastest way to get help is a backtrace.
For Linux see: https://github.com/Maproom/qmapshack/wiki/TroubleShooting#create-a-backtrace-of-a-crash-on-linux.
For Windows it would need a debug build and running QMapshack in Visual Studio.)

### Desktop
[comment]: # (please complete the following information)

 - OS: Linux Mint 20.3
 - QMapShack Version:  QMapShack_1.16.1-x86_64_Ubuntu-18.04.AppImage
 - Window Manager:  Cinnamon

### Additional context
[comment]: # (Add any other context about the problem here)
Loaded the same map in GPXSee and it rendered correctly there, so that may indicate that the issue is in QMapShack and not in the map file itself.

**Comment by ventoux84** [comment]
indeed, it is the same behavior with the cliffs of Calanques of Marseille.
The low point seems to be on the land and not towards the sea.

![Sélection_001](https://user-images.githubusercontent.com/56974650/166920072-c72d768f-2704-4482-a5f8-f0aa30ed6199.png)

**Comment by wthaem** [comment]
The attached snapshots show a region with a cliff rendered in 2 different OSM-based gmapsupp maps (left: Freizeitkarte DE, right: velomap DE).

Rendering in the left part is wrong, while rendering in the right part is correct.

In my opinion, this indicates that the wrong rendering is caused by the map data and not by QMS.

**Comment by wthaem** [comment]
I apologize for closing the issue by chance. Can, please, someone with necessary rights re-open it?

**Comment by kiozen** [comment]
Sure :smile:
